build.sh

```diff
@@ -85,7 +85,7 @@ for PKG_PATH in $(catkin_topological_order --only-folders); do
   (
   cd "$PKG_PATH"
   test -f debian/changelog && GENERATED_DEBIAN_PACKAGE=$(head -n1 debian/changelog | cut -d' ' -f1)
-  if [ -n "$CONTINUE_PACKAGE_GENERATION" ] && [ -n "$GENERATED_DEBIAN_PACKAGE" ] && stat -t "${APT_REPO}/${GENERATED_DEBIAN_PACKAGE}_"*.deb ]; then
+  if [ -n "$CONTINUE_PACKAGE_GENERATION" ] && [ -n "$GENERATED_DEBIAN_PACKAGE" ] && ls "${APT_REPO}/${GENERATED_DEBIAN_PACKAGE}_"*.deb >/dev/null 2>&1; then
```

Comment:
@jspricke regarding POSIX-ness, I've replaced `stat` with `ls`, this should be roughly equivalent, let me know if you can think of any unintentional side effects

Reply:
Ha, I guess I was too tired last night, thx.

Reply:
No problem, glad we found a way :slightly_smiling_face: